.github/workflows/tests.yml

```diff
@@ -107,13 +105,16 @@ jobs:
               core.setOutput('timeout-minutes', '60')
             }
 
+            const container = {}
             if (label_names.includes('CI-linux-wheezy')) {
               console.log('CI-linux-wheezy label found. Using Linux Debian 7 (Wheezy) container.')
-              core.setOutput('container', 'homebrew/debian7:latest')
+              container.image = 'homebrew/debian7:latest'
             } else {
               console.log('No CI-linux-wheezy label found. Using default Homebrew (Ubuntu 16.04) container.')
-              core.setOutput('container', 'ghcr.io/homebrew/ubuntu16.04:master')
+              container.image = 'ghcr.io/homebrew/ubuntu16.04:master'
             }
+            container.options = '--user=linuxbrew --workdir=/github/home'
```

Comment:
This doesn't work because of https://github.com/actions/runner/issues/878.

Reply:
A global `working-directory` for the job could work (`/github/home` on Linux, `{{ github.workspace }}` elsewhere).

Or just tweak `setup-homebrew` I suppose since it only applies to this repo anyway.

Reply:
Done. Thanks.